### DISCUSSION TITLE
Master container not showing updates?

### DISCUSSION THREAD

**Discussion by Phil-Barker** [nextcloud/all-in-one#1885] [Questions] [answered]
I run AIO on Unraid.
The master container is always showing as update required. I ignore it because the documentation states the master container will show updates through the admin panel.
It has previously shown updates needed to child containers and master container, but it doesn't seem accurate.

Currently the image on my unraid for AIO is 
`nextcloud/all-in-one                    latest       35049b54ac64   2 weeks ago     692MB`

According to docker hub the latest tag has a digest of `96c1da19c304` and was pushed 8 days ago.

I am on v4.2.0 which seems to be the latest - but not the latest docker image, and unraid keeps flagging warnings saying it needs to be updated.

Should I just continue ignoring unraid? Or should I update it to the latest docker image? Or should AIO be telling me there's an update and it isn't?

It's not serious, just confusing and the alerts about out of date images are frustrating :)

Thanks

**Comment by szaimen** [comment, marked as answer]
v4.2.0 is indeed the latest version so not sure why unraid is reporting that updates are available. Maybe their update check is somehow broken?

**Reply by Phil-Barker** [reply]
I bit the bullet and ran the upraid update on the main container... as I was swapping out motherboard today so thought might aswell when bringing it all back up..

![unraid-nextcloud-update](https://user-images.githubusercontent.com/4199779/216777343-0242f594-103a-47c9-9b15-d1bb53283a41.png)

It definitely seems like a bug as it reported nothing to do - but now shows in the UI as up to date.. Very weird but thanks for the response

**Comment by GUNT0815** [comment]
Would you mind posting your unraid/nextcloud-aio docker-compose.yml file here? Is there any special in the dc.yml for unraid? Or can just the standard config be used? As I want to transfer my nextcloud-aio from a dedicated docker-server to unraid: any suggestion how to do this step-by-step?

**Reply by Phil-Barker** [reply]
Hi,

I haven't used a docker compose file - I've used the `nextcloud-aio-mastercontainer` app in Unraid.

**Comment by Phil-Barker** [comment]
As an update to this

AIO shows all containers running and up-to-date
![image](https://user-images.githubusercontent.com/4199779/217775625-0574f9d4-3f55-4397-ad42-33c2723e57d2.png)

Unraid now shows the master container as up-to-date, since clicking update and it confirming there was nothing to do.
![image](https://user-images.githubusercontent.com/4199779/217775723-3e0cc98f-8a94-4338-a820-62f156d66a56.png)

But all the child containers permanently show as update ready
![image](https://user-images.githubusercontent.com/4199779/217775805-5c1bcd43-931b-4cde-b145-0bc92ddb47e3.png)

If you try to update any of them through unraid you get `Configuration not found. Was this container created using this plugin?` 

I think the best situation you can end up with if using Nextcloud AIO and Unraid is to mark all child-container update notifications as ignored so you don't get a popup all the time, and then just ignore the `update ready` on the docker page.
It really is a shame that Unraid doesn't better handle docker containers created outside of it's UI.

**Comment by DcryptedCA** [comment]
Same issue here, Unraid showing "update ready" even though containers are updated (via the mastercontainer).

**Comment by Cjkeenan** [comment]
Has anyone been able to solve this issue? This paired with docker update notification results in a ton of spam. Alternatively, does anyone know how to ignore certain docker containers from sending out update notifications?

**Comment by Phil-Barker** [comment]
Unfortunately not.

Unraid AIO really doesn't play well with Unraid.
The master container and child containers permanently show updates ready, and if you enable update notifications will spam you daily.

I don't think this is really a nextcloud issue at all - the containers work fine and the internal update process works fine.
Unraid needs a plugin or setting to allow exclusion of some containers from the update scans, but I looked all over and couldn't find anything.

**Reply by bozzio83** [reply]
Any luck 10 months later?

**Reply by Phil-Barker** [reply]
Nope. Still the same.

Unraid permanently says they have an upgrade available, if you upgrade it says nothing to do.
Just had to add them all to the ignore list so I don't get notifications daily. It's quite annoying but doesn't seem like there's any solution.

**Comment by Phil-Barker** [comment]
I thought I'd properly close this off

Unraid 7.0 has FAR better support for 3rd party docker containers. So it now correctly shows them as 3rd party and no update required.
I didn't see any mention in the 7.0 changes but it's a very welcome improvement

![image](https://github.com/user-attachments/assets/e10db9d3-17a6-4c4e-9e10-98c547b45e48)

**Reply by leprechaun17** [reply]
I am running Unraid 7.0.0 and have the same issue as originally mentioned in this thread. Let's see if someone can ultimately fix this.
